src/goto-symex/symex_other.cpp

```diff
@@ -137,9 +137,9 @@ void goto_symext::symex_other(
     // 3. build an assignment where the type on lhs and rhs is:
     // - array_copy: the type of the first array (even if the second is smaller)
     // - array_replace: the type of the second array (even if it is smaller)
-    DATA_INVARIANT(
-      code.operands().size() == 2,
-      "array_copy/array_replace takes two operands");
+  	DATA_INVARIANT(
+  		code.operands().size() == 2,
+  		"expected statement to have two operands at this point");
```

Comment:
This seems like a change that gives the error message less information... Why change this INVARIANT ?